tools/add_triage_signature.py

```diff
@@ -412,45 +478,58 @@ def get_logs_url_from_issue(issue):
 def get_all_triage_tickets(jclient, only_recent=False):
     recent_filter = "" if not only_recent else 'and created >= -31d'
     query = 'project = MGMT AND component = "Assisted-installer Triage" {}'.format(recent_filter)
-    idx = 0
-    block_size = 100
-    issues = []
-    while True:
-        i = jclient.search_issues(query, maxResults=block_size, startAt=idx)
-        if len(i) == 0:
-            break
-        issues.extend(i)
-        idx += block_size
+
+    return jclient.search_issues(query, maxResults=None)
+
+
+def get_issues(jclient, issue, recent_issues):
+    if issue is None:
```

Comment:
simpler workflow without saving `issues` variable
```python
    if issue:
       logger.info(f"Fetching just {issue}")
       return [get_issue(jclient, issue)]
    
    logger.info(f"Fetching {['all', 'recent'][recent_issues]} issues, this may take a while")
    return get_all_triage_tickets(jclient, only_recent=recent_issues)
```